src/base/bittorrent/torrenthandle.h

```diff
@@ -447,12 +447,15 @@ namespace BitTorrent
 
         enum StartupState
         {
-            NotStarted,
+            Initialized,
+            PreparingPaused,
+            PreparingResumed,
+            CheckingPaused,
             Starting,
-            Started
+            Working
         };
 
-        StartupState m_startupState = NotStarted;
+        StartupState m_processingState = Initialized;
```

Comment:
IMO processing state is vague, why not the original name?

Reply:
Now it contains not only "startup" states, but also other "extra" states (e.g. CheckingPaused).